live-images/index.md: change voice and layout of information (#70)
void-linux/void-docs: src/installation/live-images/index.md

# Live Installers

Void provides live installer images containing a base set of utilities, an
installer program, and package files to install a new Void system. These live
images are also useful for repairing a system that is not able to boot or
function properly.

There are `x86_64` images for both `glibc` and `musl` based systems. There are
also images for `i686`, but only `glibc` is supported for this architecture.
Live installers are not provided for other architectures. Users of other
architectures will need to use rootfs tarballs, or perform an installation
manually.

## Installer images

Void releases two types of images: base images and "flavor" images. Linux
beginners are encouraged to try one of the more full-featured flavor images, but
more advanced users may often prefer to start from a base image to install only
the packages they need.

### Base images

The base images provide only a minimal set of packages to install a usable Void
system. These base packages are only those needed to configure a new machine,
update the system, and install additional packages from repositories.

### Flavor images

> Void also provides several "flavor" images which each include a full desktop

Reviewer: We've already called out that Void provides flavor images, so we can just skip directly to describing them.
```suggestion
Each of the Void "flavor" images includes a full desktop
```

environment, web browser, and basic applications configured for that
environment. The only difference from the base images is the additional

> packages/services installed.

Reviewer: Try to favor "and" and "or" over slashes.
```suggestion
packages and services installed.
```

The install process for each of the flavor images is the same as the base
images, except that you **must** select the `Local` source when installing,
because the `Network` install only installs the base system, pulled fresh from

> the official mirrors.

Reviewer: Maybe something like this, since otherwise this paragraph is one long sentence.

```
The install process for each flavor image is the same as the base images, except
you **must** select the `Local` source when installing. If you select `Network`
instead, this only installs the base system, pulled fresh from the official
mirrors, without any additional packages.
```
